### PR TITLE
Bluetooth testing bsim: Add BT split stack tests support and some tests in CI

### PR DESCRIPTION
**Pull request by aescolar**
* Add support for BT split stack tests in the bsim bt test scripts. That is, now we have the option to build using sysbuild
* Minor improvement to the bsim bt tests run_parallel script to allow providing a file with search paths for tests
* For 2 tests, add sysbuild build files to be able to do a BT stack build for the nrf5340bsim with the controller in the netcore and the app and host in the application core.
* For the bsim CI workflow, run also the selected BT split stack tests (on an nrf5340bsim). Also move the list of which BT tests are build and run for each type to tests/bsim/bluetooth/tests.* .

Note: Adding support for the split builds with sysbuild does not affect the ability to build without sysbuild, either for the nrf52bsim or other targets. It remains possible to use the compile and run_parallel scripts as before with the nr52bsim or out of tree boards.

### CODE REVIEW

**Review by cvinayak — APPROVED** [review]

File: tests/bsim/bluetooth/compile.nrf5340bsim_nrf5340_cpuapp.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+# Copyright 2018 Oticon A/S
```
Comment (Thalley) [inline tests/bsim/bluetooth/compile.nrf5340bsim_nrf5340_cpuapp.sh:2]: Is this the correct copyright?

File: tests/bsim/bluetooth/compile.nrf5340bsim_nrf5340_cpunet.sh [hunk 2]
```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+# Copyright 2018 Oticon A/S
```
Comment (Thalley) [inline tests/bsim/bluetooth/compile.nrf5340bsim_nrf5340_cpunet.sh:2]: Is this the correct copyright?

File: tests/bsim/bluetooth/compile.nrf5340bsim_nrf5340_cpuapp.sh (hunk 1 above)
Comment (aescolar) [reply tests/bsim/bluetooth/compile.nrf5340bsim_nrf5340_cpuapp.sh:2]: Well.. the file started as a `cp` of `compile.sh`, so I tend to not change copyrights when I copy and modify lightly a file.
The whole file is so trivial it feels irrelevant to add an extra copyright line.

Comment (Thalley) [reply tests/bsim/bluetooth/compile.nrf5340bsim_nrf5340_cpuapp.sh:2]: Fair :)

**Review by Thalley — APPROVED** [review]

**Pull request merged**